packages/vega-loader/src/loader.js

```diff
@@ -89,7 +89,7 @@ async function sanitize(uri, options) {
   // should we load from file system?
   loadFile = (isFile = uri.startsWith(fileProtocol))
     || options.mode === 'file'
-    || (options.mode !== 'no-cors' && options.mode !== 'cors' && options.mode !== 'same-origin')
+    || (options.mode !== 'http')
```

Comment:
Makes sense

Reply:
That's what it was before my changes yesterday. Thanks for helping me dig into this!